data/books.json

```diff
@@ -327,5 +327,12 @@
 		"title": "The Alchemist",
 		"author": "Paulo Coelho",
 		"year": 1988
+  },
+  {
+		"quote": "If your regime can not resist jokes, then you have no regime",
```

Comment:
Im going to have to say no to this one sorry :(

Reply:
ukkhh what for?